src/components/simulation/BoxStep3.vue

```diff
@@ -0,0 +1,32 @@
+<script lang="ts" setup>
```

Comment:
Component BoxStep2 and BoxStep3 are almost the same.
Can you create a generic component named for exemple 'BoxStep' and use it in BoxStep2 and BoxStep3 please
This avoids duplicating the code